flashinfer/moe_ep/kernel_src/cutedsl_megamoe/src/moe_nvfp4_swapab/mega_runner.py

```
@@ -2182,8 +2038,8 @@ def _region(ws, by_name, offsets, name, torch_dtype):
                 print(f"  first 16 extras (src_rank, src_token, src_topk): {sample}")
         else:
             print(
-                "---- expected-vs-actual reverse check skipped "
-                "(fc1_output peek unavailable) ----"
+                f"---- expected-vs-actual reverse check skipped "
```

Ruff (0.15.21)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)